server/index.js

```diff
@@ -1,4 +1,6 @@
 const express = require('express');
+require('dotenv').config({path: __dirname + '/.env'})
```

Comment:
can be just
`require('dotenv').config();`

Reply:
That didn't work for me. This answer helped me https://stackoverflow.com/a/43522068